lib/src/component_declaration/flux_component.dart

```diff
@@ -151,33 +151,62 @@ abstract class _FluxComponentMixin<TProps extends FluxUiProps> implements Batche
   /// These subscriptions are canceled when the component is unmounted.
   List<StreamSubscription> _subscriptions = [];
 
-  // This is private and called by classes to work around super-calls not being supported in mixins
```

Comment:
[Whitespace-agnostic diff](https://github.com/Workiva/over_react/pull/195/files?w=1) in this file may help when reviewing